src/main/java/eu/dety/burp/joseph/attacks/invalid_curve/InvalidCurveOracle.java

```diff
@@ -0,0 +1,52 @@
+package eu.dety.burp.joseph.attacks.invalid_curve;
+
+import burp.IBurpExtenderCallbacks;
+import burp.IExtensionHelpers;
+import eu.dety.burp.joseph.attacks.invalid_curve.gui.InvalidCurveTableEntry;
+import eu.dety.burp.joseph.utilities.Logger;
+import org.simmetrics.StringMetric;
+import org.simmetrics.metrics.StringMetrics;
+
+/**
+ * Invalid Curve Oracle is used to compare responses if they match the valid reference.
+ * 
+ * @author Vincent Unsel
+ * @version 1.0
+ */
+public class InvalidCurveOracle {
+    private static final Logger loggerInstance = Logger.getInstance();
+    private IExtensionHelpers helpers;
+    private static final double COMPARE_THRESHOLD = 0.9;
```

Comment:
Would it not be effective to make this threshold configurable by the user? ...otherwise, this could become impractical

Reply:
...depending on the original message properties